.github/workflows/build.yml

```
@@ -34,7 +34,7 @@ jobs:
           - {os: ubuntu-22.04, ruby: '3.0', db: mariadb10.6}
           - {os: ubuntu-20.04, ruby: '2.7', db: mariadb10.6}
           - {os: ubuntu-20.04, ruby: '2.7', db: mysql80}
-          - {os: ubuntu-18.04, ruby: '2.7', db: mysql57}
+          # - {os: ubuntu-18.04, ruby: '2.7', db: mysql57} # no longer exists
```

```suggestion
          # - {os: ubuntu-20.04, ruby: '2.7', db: mysql57} # no longer exists
```
To continue testing MySQL 5.7 let's see if it'll run on 20.04